api/v1alpha2/linodecluster_types.go

```diff
@@ -104,7 +104,8 @@ func (lm *LinodeCluster) SetConditions(conditions clusterv1.Conditions) {
 // NetworkSpec encapsulates Linode networking resources.
 type NetworkSpec struct {
 	// LoadBalancerType is the type of load balancer to use, defaults to NodeBalancer if not otherwise set
-	// +kubebuilder:validation:Enum=NodeBalancer;dns
+	// +kubebuilder:validation:Enum=NodeBalancer;dns;external
+	// +kubebuilder:default=NodeBalancer
```

Comment:
Added the ++kubebuilder:default=NodeBalancer, which was missing. Not sure if it was done on purpose.